### PR TITLE
Apply stereo to setup for pick/stow.launch

### PR DESCRIPTION
**Pull request by YutoUchimi**

### CODE REVIEW

**Comment by wkentaro** [comment]
Please rebase from https://github.com/start-jsk/jsk_apc/pull/2233

File: jsk_arc2017_baxter/launch/setup/setup_for_pick.launch [hunk 1]
```diff
@@ -11,81 +11,162 @@
...
-        max_queue_size: 100
-      </rosparam>
-    </node>
-    <include file="$(find jsk_arc2017_common)/launch/object_segmentation_3d.launch">
-      <arg name="NODELET_MANAGER" value="$(arg manager_right)" />
-      <arg name="INPUT_IMAGE" value="/right_hand_camera/rgb/image_rect_color"/>
-      <arg name="INPUT_CLOUD" value="extract_indices_target_bin/output"/>
-    </include>
-  </group>
+  <group if="$(arg stereo)">
```
Comment (wkentaro) [inline jsk_arc2017_baxter/launch/setup/setup_for_pick.launch]: Why you need this group?
Why don't you do like below?

```
<arg name="INPUT_CLOUD" value="/right_hand_camera/depth_registered/points" unless="$(arg stereo)" />
<arg name="INPUT_CLOUD" value="/right_hand_camera/fused/depth_registered/points" if="$(arg stereo)" />
<node ..
  <remap name="~input" value="$(arg INPUT_CLOUD)" />
</node>
```

**Comment by YutoUchimi** [comment]
> Why you need this change?
I mean the last commit db65c0b

When I launched the program, I saw error messages on AttentionClipper.
I suspended then it was because the namespace of nodelet manager was wrong.
But I found the actual ploblem was I had used old fc.rosinstall.

Now I discarded that change.

**Comment by wkentaro** [comment]
LGTM. I appreciate this contribution, and it's really nice work.

**Comment by wkentaro** [comment]
I will rebase because of some conflicts.
image

**Pull request merged**